### PR TITLE
yashim/perf: specify higher compression for certain images. force avif as main format

### PR DESCRIPTION
**Pull request by yashim-deriv**
Changes:

-   ...

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [x] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by mohsen-deriv — APPROVED** [review]

File: src/pages/home/_platform-slideshow.tsx [hunk 1]
```diff
@@ -68,34 +30,87 @@ const StyledImage = styled(QueryImage)<{ $is_hidden: boolean }>`
...
-                { key: 'hero3', image: data.hero_platform3 },
-                { key: 'hero4', image: data.hero_platform4 },
+                {
+                    key: 'hero1',
+                    image: (
+                        <StaticImage
+                            src="../../images/common/home/hero_platform1.png"
+                            alt="mobile app deriv go"
+                            formats={['avif', 'webp', 'auto']}
+                            quality={36}
```
Comment (ali-hosseini-deriv) [inline src/pages/home/_platform-slideshow.tsx:45]: why the quality is 36 here and the rest are 40?
is it intended?

File: src/pages/home/_platform-slideshow.tsx [hunk 2]
```diff
@@ -16,45 +14,9 @@ const ImagePlaceHolder = styled.div`
...
-    }
-`
-
-const StyledImage = styled(QueryImage)<{ $is_hidden: boolean }>`
+const ImageWrapper = styled.div<{ $is_hidden: boolean }>`
     opacity: ${({ $is_hidden }) => ($is_hidden ? '0' : '1')};
     display: ${({ $is_hidden }) => ($is_hidden ? 'none' : 'block')};
-    animation: fade 1s ease-in-out;
-
-    @media ${device.tabletL} {
```
Comment (ali-hosseini-deriv) [inline src/pages/home/_platform-slideshow.tsx:47]: is this related to the improvement that you made?
i mean not disabling animation can be different from the thing that is in the scope of this task.

Comment (yashim-deriv) [reply src/pages/home/_platform-slideshow.tsx:47]: the animation adds to UX feel of longer loading. I will add it back if any of the QA complains 😆

Comment (ali-hosseini-deriv) [reply src/pages/home/_platform-slideshow.tsx:47]: correct

**Review by ali-hosseini-deriv — DISMISSED** [review]

**Review by habib-deriv — APPROVED** [review]
LGTM

**Review by mohsen-deriv — DISMISSED** [review]

**Review by habib-deriv — APPROVED** [review]
LGTM
